diff -U 3 -H -d -r -N -- wxWidgets-2.5.3-orig/utils/configtool/src/appsettings.cpp wxWidgets-2.5.3/utils/configtool/src/appsettings.cpp
--- wxWidgets-2.5.3-orig/utils/configtool/src/appsettings.cpp	2005-01-10 12:59:51.950473496 +0100
+++ wxWidgets-2.5.3/utils/configtool/src/appsettings.cpp	2005-01-10 12:39:34.960484096 +0100
@@ -29,6 +29,7 @@
 #include "wx/effects.h"
 #include "wx/spinctrl.h"
 #include "wx/tooltip.h"
+#include "wx/wx.h"
 
 #endif
 
diff -U 3 -H -d -r -N -- wxWidgets-2.5.3-orig/utils/configtool/src/configitemselector.cpp wxWidgets-2.5.3/utils/configtool/src/configitemselector.cpp
--- wxWidgets-2.5.3-orig/utils/configtool/src/configitemselector.cpp	2005-01-10 12:59:51.738505720 +0100
+++ wxWidgets-2.5.3/utils/configtool/src/configitemselector.cpp	2005-01-10 12:41:12.717622752 +0100
@@ -27,6 +27,7 @@
 #include "wx/scrolwin.h"
 #include "wx/spinctrl.h"
 #include "wx/spinbutt.h"
+#include "wx/wx.h"
 
 #endif
 
diff -U 3 -H -d -r -N -- wxWidgets-2.5.3-orig/utils/configtool/src/mainframe.cpp wxWidgets-2.5.3/utils/configtool/src/mainframe.cpp
--- wxWidgets-2.5.3-orig/utils/configtool/src/mainframe.cpp	2005-01-10 12:59:51.682514232 +0100
+++ wxWidgets-2.5.3/utils/configtool/src/mainframe.cpp	2005-01-10 12:43:38.687431968 +0100
@@ -23,6 +23,7 @@
 #ifndef WX_PRECOMP
 
 #include "wx/splitter.h"
+#include "wx/wx.h"
 
 #endif
 
diff -U 3 -H -d -r -N -- wxWidgets-2.5.3-orig/utils/configtool/src/settingsdialog.cpp wxWidgets-2.5.3/utils/configtool/src/settingsdialog.cpp
--- wxWidgets-2.5.3-orig/utils/configtool/src/settingsdialog.cpp	2005-01-10 12:59:51.901480944 +0100
+++ wxWidgets-2.5.3/utils/configtool/src/settingsdialog.cpp	2005-01-10 12:44:48.856764608 +0100
@@ -27,6 +27,7 @@
 #include "wx/scrolwin.h"
 #include "wx/spinctrl.h"
 #include "wx/spinbutt.h"
+#include "wx/wx.h"
 
 #endif
 
diff -U 3 -H -d -r -N -- wxWidgets-2.5.3-orig/utils/configtool/src/utils.cpp wxWidgets-2.5.3/utils/configtool/src/utils.cpp
--- wxWidgets-2.5.3-orig/utils/configtool/src/utils.cpp	2005-01-10 12:59:52.015463616 +0100
+++ wxWidgets-2.5.3/utils/configtool/src/utils.cpp	2005-01-10 12:45:35.351696304 +0100
@@ -31,6 +31,7 @@
 #include "wx/process.h"
 #include "wx/variant.h"
 #include "wx/cmdline.h"
+#include "wx/wx.h"
 
 #endif
 
diff -U 3 -H -d -r -N -- wxWidgets-2.5.3-orig/utils/configtool/src/wxconfigtool.cpp wxWidgets-2.5.3/utils/configtool/src/wxconfigtool.cpp
--- wxWidgets-2.5.3-orig/utils/configtool/src/wxconfigtool.cpp	2005-01-10 12:59:51.936475624 +0100
+++ wxWidgets-2.5.3/utils/configtool/src/wxconfigtool.cpp	2005-01-10 11:58:33.919619176 +0100
@@ -26,6 +26,7 @@
 #include "wx/menuitem.h"
 #include "wx/tooltip.h"
 #include "wx/variant.h"
+#include "wx/wx.h"
 
 #endif
 
diff -U 3 -H -d -r -N -- wxWidgets-2.5.3-orig/utils/HelpGen/src/HelpGen.cpp wxWidgets-2.5.3/utils/HelpGen/src/HelpGen.cpp
--- wxWidgets-2.5.3-orig/utils/HelpGen/src/HelpGen.cpp	2005-01-10 12:59:53.420250056 +0100
+++ wxWidgets-2.5.3/utils/HelpGen/src/HelpGen.cpp	2005-01-10 12:49:08.512290968 +0100
@@ -60,6 +60,7 @@
     #include "wx/log.h"
     #include "wx/dynarray.h"
     #include "wx/app.h"
+    #include "wx/wx.h"
 #endif // WX_PRECOMP
 
 #include "wx/file.h"
