--- wxWindows-2.4.2/acinclude.m4.wiget	2002-12-18 19:13:53.000000000 +0100
+++ wxWindows-2.4.2/acinclude.m4	2003-09-23 14:44:44.000000000 +0200
@@ -96,8 +96,7 @@
   if test "$cross_compiling" = "yes"; then
     ifelse([$2], , :, [$2])
   else
-    AC_LANG_SAVE
-    AC_LANG_CPLUSPLUS
+    AC_LANG_PUSH(C++)
 
     AC_CHECK_HEADERS(iostream)
 
@@ -107,7 +106,7 @@
       ifelse([$2], , :, [$2])
     fi
 
-    AC_LANG_RESTORE
+    AC_LANG_POP(C++)
   fi
 ])
 
@@ -121,8 +120,7 @@
 [
   AC_CACHE_CHECK([if C++ compiler supports bool], wx_cv_cpp_bool,
   [
-    AC_LANG_SAVE
-    AC_LANG_CPLUSPLUS
+    AC_LANG_PUSH(C++)
 
     AC_TRY_COMPILE(
       [
@@ -140,7 +138,7 @@
       ]
     )
 
-    AC_LANG_RESTORE
+    AC_LANG_POP(C++)
   ])
 
   if test "$wx_cv_cpp_bool" = "yes"; then
@@ -158,8 +156,7 @@
   AC_CACHE_CHECK([if C++ compiler supports the explicit keyword],
                  wx_cv_explicit,
   [
-    AC_LANG_SAVE
-    AC_LANG_CPLUSPLUS
+    AC_LANG_PUSH(C++)
 
     dnl do the test in 2 steps: first check that the compiler knows about the
     dnl explicit keyword at all and then verify that it really honours it
@@ -187,7 +184,7 @@
       wx_cv_explicit=no
     )
 
-    AC_LANG_RESTORE
+    AC_LANG_POP(C++)
   ])
 
   if test "$wx_cv_explicit" = "yes"; then
--- wxWidgets-2.5.3/configure.in.orig	2005-02-20 19:38:45.461014000 +0100
+++ wxWidgets-2.5.3/configure.in	2005-02-20 20:58:15.485860176 +0100
@@ -1337,8 +1337,7 @@
 dnl   adds -traditional for gcc if needed
 AC_PROG_GCC_TRADITIONAL
 
-AC_LANG_SAVE
-AC_LANG_CPLUSPLUS
+AC_LANG_PUSH(C++)
 
 dnl C++-compiler checks
 dnl   defines CXX with the compiler to use
@@ -1350,7 +1349,7 @@
 CXXFLAGS=${CXXFLAGS:=}
 AC_PROG_CXX
 
-AC_LANG_RESTORE
+AC_LANG_POP(C++)
 
 dnl ranlib command
 dnl   defines RANLIB with the appropriate command
@@ -2218,7 +2217,7 @@
             dnl HAVE_STDLIB_H which are already defined and this provokes
             dnl a compiler warning which configure considers as an error...
             AC_MSG_CHECKING(for jpeglib.h)
-            AC_CACHE_VAL(ac_cv_header_jpeglib_h,
+            AC_CACHE_VAL(ac_cv_header_jpeglib_h,[
                 AC_TRY_COMPILE(
                     [
                         #undef HAVE_STDLIB_H
@@ -2230,7 +2229,7 @@
                     ac_cv_header_jpeglib_h=yes,
                     ac_cv_header_jpeglib_h=no
                 )
-            )
+            ])
             AC_MSG_RESULT($ac_cv_header_jpeglib_h)
 
             if test "$ac_cv_header_jpeglib_h" = "yes"; then
