--- wxWidgets-2.7.1/src/html/m_image.cpp.orig	2006-10-12 14:52:20.000000000 +0200
+++ wxWidgets-2.7.1/src/html/m_image.cpp	2006-10-15 20:32:56.000000000 +0200
@@ -333,6 +333,11 @@
 
     DECLARE_NO_COPY_CLASS(wxGIFTimer)
 };
+
+static int safedelay(int ms)
+{
+       return (ms >= 10) ? ms : 10;
+}
 #endif
 
 
@@ -388,7 +393,7 @@
                         if ( m_gifDecoder->IsAnimation() )
                         {
                             m_gifTimer = new wxGIFTimer(this);
-                            m_gifTimer->Start(m_gifDecoder->GetDelay(0), true);
+                            m_gifTimer->Start(safedelay(m_gifDecoder->GetDelay(0)), true);
                         }
                         else
                         {
@@ -522,7 +527,7 @@
         win->Refresh(img.HasMask(), &rect);
     }
 
-    timer->Start(m_gifDecoder->GetDelay(m_nCurrFrame), true);
+    timer->Start(safedelay(m_gifDecoder->GetDelay(m_nCurrFrame)), true);
 }
 
 void wxHtmlImageCell::Layout(int w)
