diff -U 3 -H -d -r -N wxWidgets-2.5.4.orig/utils/configtool/src/appsettings.cpp wxWidgets-2.5.4/utils/configtool/src/appsettings.cpp
--- wxWidgets-2.5.4.orig/utils/configtool/src/appsettings.cpp	2005-02-28 15:13:40.113087000 +0100
+++ wxWidgets-2.5.4/utils/configtool/src/appsettings.cpp	2005-02-28 15:50:00.161669376 +0100
@@ -30,6 +30,7 @@
 #include "wx/spinctrl.h"
 #include "wx/tooltip.h"
 #include "wx/settings.h"
+#include "wx/wx.h"
 
 #endif
 
diff -U 3 -H -d -r -N wxWidgets-2.5.4.orig/utils/configtool/src/settingsdialog.cpp wxWidgets-2.5.4/utils/configtool/src/settingsdialog.cpp
--- wxWidgets-2.5.4.orig/utils/configtool/src/settingsdialog.cpp	2005-02-28 15:13:40.115086000 +0100
+++ wxWidgets-2.5.4/utils/configtool/src/settingsdialog.cpp	2005-02-28 15:52:36.612885176 +0100
@@ -30,6 +30,7 @@
 #include "wx/sizer.h"
 #include "wx/statbox.h"
 #include "wx/dirdlg.h"
+#include "wx/wx.h"
 
 #endif
 
diff -U 3 -H -d -r -N wxWidgets-2.5.4.orig/utils/configtool/src/utils.cpp wxWidgets-2.5.4/utils/configtool/src/utils.cpp
--- wxWidgets-2.5.4.orig/utils/configtool/src/utils.cpp	2005-02-28 15:13:40.116086000 +0100
+++ wxWidgets-2.5.4/utils/configtool/src/utils.cpp	2005-02-28 15:53:02.121007352 +0100
@@ -35,6 +35,7 @@
 #include "wx/log.h"
 #include "wx/sizer.h"
 #include "wx/icon.h"
+#include "wx/wx.h"
 
 #endif
 
